readthedocs_theme/templates/includes/plausible-events.html

```diff
@@ -0,0 +1,33 @@
+{# 
+    Enable us to track clicks just by setting a `data-analytics` attribute on links.
+    Docs: https://plausible.io/docs/custom-event-goals#code-example-for-tracking-link-clicks
+#}
+<script>
+    let links = document.querySelectorAll("a[data-analytics]");
+    for (var i = 0; i < links.length; i++) {
+        links[i].addEventListener('click', handleLinkEvent);
+        links[i].addEventListener('auxclick', handleLinkEvent);
+    }
+  
+    function handleLinkEvent(event) {
+        var link = event.target;
+        var middle = event.type == "auxclick" && event.which == 2;
+        var click = event.type == "click";
+        while (link && (typeof link.tagName == 'undefined' || link.tagName.toLowerCase() != 'a' || !link.href)) {
+            link = link.parentNode;
+        }
+        if (middle || click) {
+            let attributes = link.getAttribute('data-analytics').split(/,(.+)/);
+            let events = [JSON.parse(attributes[0]), JSON.parse(attributes[1] || '{}')];
+            plausible(...events);
+        }
+        if (!link.target) {
+            if (!(event.ctrlKey || event.metaKey || event.shiftKey) && click) {
+                setTimeout(function () {
+                    location.href = link.href;
+                }, 150);
+                event.preventDefault();
+            }
+        }
+    }
+  </script>
```

Comment:
We should move this to the standard site JS source, instead of using inline JS. I can move it real quick.

Reply:
Cool -- I thought it was maybe nicer to keep on it's own for clarity, but happy with whatever approach makes sense.

Reply:
I addressed some issues in #156 and tested that we're receiving events in Plausible